### DISCUSSION TITLE
Is there any function to mirror the entities?

### DISCUSSION THREAD

**Discussion by chibai** [mozman/ezdxf#543] [Q&A] [answered]
I want to mirror or flip the entity by some lines or just x-axis and y-axis.
I don't know if there's any function in ezdxf to do this~
Thanks

**Comment by mozman** [comment, marked as answer]
The mirror transformation is scaling by negative values. Mirror along the x-axis is ``entity.scale(-1, 1, 1)`` or ``entity.transform(Matrix44.scale(-1, 1, 1))``. Mirroring about an arbitrary line (plane) is not implemented, but creating such a transformation matrix can possibly be found using Google.

**Reply by chibai** [reply]
Got it
